shared/transforms/xccdf-addcce-remediations.xslt

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:xccdf="http://checklists.nist.gov/xccdf/1.1" xmlns:xhtml="http://www.w3.org/1999/xhtml" exclude-result-prefixes="xccdf">
+
+ <xsl:template name="find-and-replace">
+   <xsl:param name="text"/>
+   <xsl:param name="replace"/>
+   <xsl:param name="with"/>
+   <xsl:choose>
+     <xsl:when test="contains($text,$replace)">
+       <xsl:value-of select="substring-before($text,$replace)"/>
+       <xsl:value-of select="$with"/>
+       <xsl:call-template name="find-and-replace">
+         <xsl:with-param name="text" select="substring-after($text,$replace)"/>
+         <xsl:with-param name="replace" select="$replace"/>
+         <xsl:with-param name="with" select="$with"/>
+       </xsl:call-template>
+     </xsl:when>
+     <xsl:otherwise>
+       <xsl:value-of select="$text" />
+     </xsl:otherwise>
+   </xsl:choose>
+ </xsl:template>
+
+ <xsl:template match="xccdf:fix/text()">
+   <xsl:choose>
+   <xsl:when test="contains(., 'CCENUM')">
+     <xsl:call-template name="find-and-replace">
+       <xsl:with-param name="text" select="."/>
+       <xsl:with-param name="replace" select="'CCENUM'"/>
```

Comment:
Please choose a more unique token than `CCENUM`. With CCENUM it's not obvious it's going to be replaced when looking at remediation scripts. I recommend at least `$CCENUM`, should be obvious that that is a variable to be replaced.